### ISSUE TITLE
Organizational relationships

### ISSUE THREAD

**Issue by justgo129** [USGCRP/gcis-ontology#137]
Hi everyone,
  Currently we don't have a structured way through which to associate one organization to another.  For instance, let's take a look at:
https://data.globalchange.gov/organization/university-wisconsin-milwaukee.thtml

We would like to affirm that the University of Wisconsin-Milwaukee is a branch of the University of Wisconsin System.  In this example, we have captured the "University of Wisconsin (system)" component, but not the "branch of."

The "branch of" association can be found at:
https://data.globalchange.gov/organization/university-wisconsin-milwaukee.thtml

Would this require the addition of terms such as "branch of," "office of," "affiliated with," etc. in our ontology or is there a way around this?  I am thinking of the use case of someone wanting to see all of the contributions of the University of Wisconsin system to the NCA3.  This would include:
- The identification of all campuses making up the University system:
  https://data.globalchange.gov/organization/university-wisconsin-madison
- Identification of departments of the system's branches (e.g. https://data.globalchange.gov/organization/university-wisconsin-department-rural-sociology

Would we need to add the "branch of," "department of" terms to our ontology in order to facilitate the writing of this SPARQL query?

**Comment by zednis** [comment]
I believe we should evaluate the use of http://www.w3.org/TR/vocab-org/ for our organization relationship use cases.

**Comment by justgo129** [comment]
Agreed, building on the use of that ontology.  @rewolfe @aulenbac @bduggan @zednis @xgmachina @lic10, as decided during our weekly meeting, let's use this issue for adding and discussing various use cases in order to inform a final decision on this matter.

**Comment by justgo129** [comment]
Here's a second use case, which I call "investigation of extent of organization involvement."  A project PI is interested in using GCIS to explore the extent of NSF's involvement in the USG's Global Change Research portfolio.  Relevant information includes:
- studies and organizational programs funded by NSF
- organizations directly financially supported by NSF
- organizations related to or affiliated with those directly supported by NSF
- organizations with which NSF is affiliated
- research projects developed by organizations funded by NSF
- organizational programs funded by NSF
- organizations which use NSF information
- studies written by NSF staff (or commissioned by NSF), documents hosted and distributed by NSF (non-organizational relationship clearly but you get the point).

I would expect the relationships "affiliated with," "divisions of," and "funded by" to be used most frequently during the analysis.

**Comment by zednis** [comment]
@justgo129 Let's create a table (google spreadsheet?) where we can map these use cases with existing properties from GCIS, properties from the W3C org ontology, or other vocabularies already used by GCIS.  This should help us identify if any gaps exist.

**Comment by rewolfe** [comment]
This may be something we could ask for Hook's group to help with.

On Tue, Sep 8, 2015 at 1:19 PM, Stephan Zednik notifications@github.com
wrote:

> @justgo129 https://github.com/justgo129 Let's create a table (google
> spreadsheet?) where we can map these use cases with existing properties
> from GCIS, properties from the W3C org ontology, or other vocabularies
> already used by GCIS. This should help us identify if any gaps exist.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/USGCRP/gcis-ontology/issues/137#issuecomment-138638961
> .

## 

Robert Wolfe, NASA GSFC @ USGCRP, o: 202-419-3470, m: 301-257-6966

**Comment by justgo129** [comment]
Agreed.  Copying @pymonger.

**Comment by justgo129** [comment]
Per discussion on 9/22, I'm working with Rama on formally documenting this use case.  A draft has been created and will I update the group regularly.

**Comment by justgo129** [comment]
Rama and I have created a use case.  However, I'm a little unsure about how to semantically properly capture the nature of the relationship between orgs.  e.g. unlike in the [html](http://data.globalchange.gov/organization/adaptive-management-working-group.html) there is no "program of" [here](http://data.globalchange.gov/organization/adaptive-management-working-group.thtml).  How would one model this?  I didn't see anything promising at http://www.w3.org/TR/vocab-org/

**Comment by zednis** [comment]
link to the use case?

**Comment by justgo129** [comment]
Update: discussed during January 19, 2016 call.  @zednis will research further.

**Comment by zednis** [comment]
@justgo129 I did not see any qualified (n-ary) organization to organization relationships in the W3C Organization Ontology.

There is such a relationship for the membership of an agent in an organization - https://www.w3.org/TR/vocab-org/#reporting_structure, but I don't think any of your use cases require this relationship.

Looking at the bullets that you listed as 'use cases' above you suggest three properties might be of relevance

> I would expect the relationships "affiliated with," "divisions of," and "funded by" to be used most frequently during the analysis.

The usage note for [`org:linkedTo`](https://www.w3.org/TR/vocab-org/#org:linkedTo) suggests sub-properties can be used to denote specialized org-to-org relationships.

_Specializations of this can be used to, for example, denote funding or supply chain relationships._

So my guess is it would make the most sense to create `gcis:fundedBy` and `gcis:affiliatedWith` properties which are sub-properties of `org:linkedTo`.

The W3C Organization Ontology does provide guidance on how to use the ontology to model organization hierarchy - https://www.w3.org/TR/vocab-org/#organizational_structure us sub-organization relationships.  Classes for specific sub-organization classifications (division, lab, research center, etc) are left to the domain to define.

We could optionally define a property such as `gcis:divisionOf` modeled as a sub-property of `org:subOrganizationOf`, but I think we would do just as well to directly use `org:subOrganizationOf` and define a small set of subclasses of `org:Organization` that would be used to classify the organization units.

**Comment by justgo129** [comment]
I agree with your rationale, @zednis.  Just to be sure I understand though, since subOrganizationOf has domain org:Organization, it would still be accurate to consider suborganizations as separate but related organizations.  e.g.
http://data.globalchange.gov/organization/university-washington-department-civil-environmental-engineering could still be an org distinct from http://data-stage.globalchange.gov/organization/university-washington , with both being "organizations" in-lieu of one being an org and one being a "suborg."

If @rewolfe agrees, I think we can create:
gcis:fundedBy
gcis:affiliatedWith
gcis:branchOf
gcis:councilOf
gcis:centerOf (forthcoming in the db - to describe interdisciplinary research centers at universities)

and represent  "officeOf," "departmentOf" and "divisionOf" through subOrganizationOf.  As long as we list "officeOf" in the html we should be all right given legal definitions.

**Comment by zednis** [comment]
- Why define branch, council, and center sub-properties but not office, department, and division?
- Is `gcis:councilOf` a org-to-org relationship?
- funded and affiliated would be sub-properties of `org:linkedTo`, the others would be sub-properties of `org:subOrganizationOf`

**Comment by justgo129** [comment]
sure.  @zednis: an example of councilOf is 
http://data.globalchange.gov/organization/presidents-council-advisors-science-technology
We also have "programOf" btw.

You have a good point.  I'm not so sure "branchOf" should be a suborganization though.  I'm thinking of branchOf in the context of the branch of a bank, or the branch of a larger office.  As such, it's more emblematic of an org (as opposed to a suborg) since it has the functions of a larger entity.

A great example showing the various types of the relationships and the ways in which we use them is available [here](http://data.globalchange.gov/organization/us-geological-survey).

**Comment by zednis** [comment]
I can see a lot of thought has already gone into potential changes but I don't think we have explored all use cases in detail.

I think we should proceed by creating a pull request as a proposal and then discussing the proposed change and various use cases in depth till we gave agreement (or at least consensus).

**Comment by justgo129** [comment]
As an update, we discussed this on 2/2.  As this is a work in progress, I'll keep #137 open until a pull request is made and merged.
